src/utils/useDidMountEffect.js

```diff
@@ -0,0 +1,13 @@
+import { useRef, useEffect } from 'react';
+
+export const useDidMountEffect = (func, deps) => {
+    const didMount = useRef(false);
+
+    useEffect(() => {
+        if (didMount.current) {
+            func();
+        } else {
+            didMount.current = true;
+        }
+    }, deps);
```

Comment:
Wouldn't it be better to have this in TS so we can check that `deps` is indeed an array?

Also, I think something along the lines of `useSkipEffectOnMount` would better describe what the hook does.

For the sake of being generic, I would add a third optional `cleanup` argument to be returned by the `useEffect`.